README.md

```diff
@@ -74,4 +77,35 @@ specified by the `RUN_MODE` environment variable:
 
 When shutting down, you should always prefer to send the `SIGINT` signal instead of `SIGKILL` so
 the service has time to finish any pending background work and all dependencies are gracefully
-disconnected.
\ No newline at end of file
+disconnected.
+
+## Bugs and feature requests
+
+If you encounter a bug or have a feature request, we encourage you to follow the steps below:
+
+ 1. **Search for existing issues:** Before submitting a new issue, please search [existing and closed issues](../../issues) to check if a similar problem or feature request has already been reported.
```

Comment:
Can you make sure these lines do not exceed 100 chars per line? There's a very handy VScode plugin I use called Rewrap that does this with 1 keystroke

Reply:
Done.